spacy/tests/tokens/test_tokens_api.py

```diff
@@ -208,6 +208,7 @@ def test_has_vector(EN):
     assert doc.has_vector
 
 
+@pytest.mark.models
```

Comment:
Thanks for your work on this!

Instead of needing models, could you set the heads, dependencies and tags using the `doc.from_array` method? I'm trying to progressively eliminate the tests that need the models.